i18n/id/white-paper.md

```diff
@@ -0,0 +1,361 @@
+
+Sayangnya, Log4j bukanlah satu-satunya contoh. core-js diunduh 30 juta kali per minggu sebagai dasar dari setiap aplikasi Node.js, namun juga hampir tidak didanai. Baru-baru ini beberapa pengembang inti bitcoin mengundurkan diri, dengan alasan, di antara alasan-alasan lainnya, *kurangnya kompensasi finansial* atas keputusan mereka.
+
+Sudah ada beberapa upaya untuk menyediakan struktur insentif, biasanya melibatkan sistem dukungan dan hadiah. Dukungan memungkinkan konsumen sumber terbuka untuk menyumbang ke proyek-proyek yang mereka sukai. Namun, bayangkan sumber terbuka sebagai menara batu bata di mana lapisan bawahnya sudah lama dilupakan, tetapi masih dipertahankan oleh para insinyur yang berdedikasi dan diandalkan oleh lebih banyak pengembang. Hanya proyek-proyek di puncak menara yang biasanya dikenal dan menerima dukungan.
+Pemilihan yang bias ini menyebabkan batu bata penting yang menopang menara tidak menarik sumbangan, sementara batu bata favorit menerima lebih banyak dari yang mereka butuhkan.
+Hadiah memungkinkan konsumen proyek untuk mengusulkan pembayaran bagi pengembang untuk membangun fitur tertentu, sehingga hanya memberi imbalan kepada proyek untuk melakukan hal-hal yang belum tentu demi kepentingan terbaik mereka. Dan sekali lagi, hanya memberi penghargaan kepada proyek favorit.
+
+Dalam makalah ini, kami mengusulkan tea - sistem terdesentralisasi untuk memberikan remunerasi yang adil kepada pengembang sumber terbuka berdasarkan kontribusi mereka terhadap seluruh ekosistem dan diberlakukan melalui algoritme insentif tea yang diterapkan di semua entri dalam registri tea.
+
+[Tampilan sederhana dari sistem imbalan seduhan tea.] (img/figure-1.svg)
```

Comment:
Please add `!` in front of the [] to indicate it's an image and no spaces between `[]()`